### PR TITLE
jurin_equation

### PR DESCRIPTION
**Pull request by Darya2909**

### CODE REVIEW

File: symplyphysics/laws/hydro/jurin_equation.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+# Description
+## The Jurin equation determines the height of the liquid rise in the capillaries.
+
+
+## Law: h = 2 * a * cos(theta) / (⍴ * g * r)
+## Where:
+## h is the height of the liquid column
+## a is the coefficient of surface tension of the liquid
+## theta is the angle of liquid wetting of the capillary wall
```
Comment (blackyblack) [inline symplyphysics/laws/hydro/jurin_equation.py:13]: Explain this angle a bit more. There are various options how to measure it.

File: symplyphysics/laws/hydro/jurin_equation.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    density_of_liquid_: Quantity, radius_: Quantity) -> Quantity:
+    result_expr = solve(law, height, dict=True)[0][height]
+    result_height = result_expr.subs({
+        surface_tension_coefficient: surface_tension_coefficient_,
+        angle: angle_,
+        density_of_liquid: density_of_liquid_,
+        radius: radius_,
+    })
+    if Quantity(result_height).scale_factor < radius_.scale_factor:
+        raise ValueError("The height must be greater than the radius")
```
Comment (blackyblack) [inline symplyphysics/laws/hydro/jurin_equation.py:48]: `greater or equal`
Also add numbers you've got to the exception.

Comment (Darya2909) [reply symplyphysics/laws/hydro/jurin_equation.py:48]: What numbers?

Comment (blackyblack) [reply symplyphysics/laws/hydro/jurin_equation.py:48]: Scale factors that you're comparing.

File: symplyphysics/laws/hydro/jurin_equation.py [hunk 3]
```diff
@@ -44,6 +44,6 @@ def calculate_height(surface_tension_coefficient_: Quantity, angle_: Quantity,
         density_of_liquid: density_of_liquid_,
         radius: radius_,
     })
-    if Quantity(result_height).scale_factor < radius_.scale_factor:
+    if Quantity(result_height).scale_factor <= radius_.scale_factor:
```
Comment (blackyblack) [inline symplyphysics/laws/hydro/jurin_equation.py:47]: Don't throw when they are equal.

File: symplyphysics/laws/hydro/jurin_equation.py (hunk 2 above)
Comment (Darya2909) [reply symplyphysics/laws/hydro/jurin_equation.py]: I don't quite understand what you mean, I'm throwing an exception anyway.

Comment (blackyblack) [reply symplyphysics/laws/hydro/jurin_equation.py]: Yes, but imagine you're a user. You've got an exception but you do not see actual numbers to compare and you do not know how to fix it. So I suggest to add numbers to exception message.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**